examples/cs/comm/pingpong/PingPong.cs

```diff
@@ -31,7 +32,9 @@ public static void Main()
 
             Shutdown(transport);
 
-            Console.WriteLine("Done with all requests.");
+            // Let everything else finish so we can print last.
+            Thread.Sleep(50);
```

Comment:
Doesn't solve the problem. The writes I'm trying to wait for are happening in the metrics handler, which are being triggered asynchronously, so it's not that surprising that this doesn't help.

This is only an issue because we're running an entire world in one process and care about the order things make it to the console. I'd like to leave it as is.